server/sandbox_run_linux.go

```diff
@@ -96,6 +96,12 @@ func (s *Server) runPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
 		}
 	}()
 
+	securityContext := req.GetConfig().GetLinux().GetSecurityContext()
+	if securityContext == nil {
+		return nil, fmt.Errorf("no security context found in config.")
```

Comment:
this was a warning but now we're erroring out, I don't think it's an error to not have selinux configuration

Reply:
Actually looking at the code, if GetSecurityContext() returns nil, all sorts of stuff will blow up.
Should we just assume it is not nil, and crash if it is?

Reply:
it's sort of guaranteed from the kubelet to be not nil that's why we just log and let it crash